jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -384,8 +406,24 @@ void update_payloadOmitsId_noIdMissMatchErrorThrown() {
     @Test
     void update_payloadViolatesEntityConstraints_resourceConstraintViolationExceptionThrown() {
         var controller = getResourceApiController(DummyEntityD.class);
-        assertThatExceptionOfType(ResourceConstraintViolationException.class).isThrownBy(() -> controller.update(TENANT_ID, NON_EXISTENT_ID, "{}"))
-                .withMessageContainingAll("description", "telephone", "has the following error(s):");
+        Throwable thrown = catchThrowable(() -> controller.create(TENANT_ID, "{}"));
```

Comment:
This test was used to test the validation triggers on the update endpoint. This is now testing on create which is covered in the other one. 
Can we follow the pattern of the other update tests to update a resource.

Reply:
Missed this, should be fixed now